ReactCommon/yoga/Yoga.podspec

```diff
@@ -27,7 +27,7 @@ Pod::Spec.new do |spec|
   spec.authors = 'Facebook'
   spec.source = source
 
-  spec.module_name = 'yoga'
+  spec.module_name = 'Yoga'
   spec.header_dir = 'yoga'
```

Comment:
I have no special knowledge here but it seems suspicious to have header_dir lowercase while module name is upper? Makes the spidey-sense tingle but it may actually be lower for the files :shrug:

Reply:
yeah tbh I was wondering if we should change both

Reply:
but wanted to get CI to run first, and get Riccardo's take ;)

Reply:
The CI is red btw, and it seems related to this change